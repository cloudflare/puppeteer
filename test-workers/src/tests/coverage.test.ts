/**
 * @license
 * Copyright 2025 Google Inc.
 * SPDX-License-Identifier: Apache-2.0
 */
import {describe, it} from 'vitest';

describe('browser does basic manipulations', () => {
  it('gets the title of google developers website', async ({expect}) => {
    await new Promise(resolve => {
      setTimeout(resolve, 30000);
    });
    const response = await fetch(
      `https://puppeteer-test-worker.cloudflare-browser-rendering-085.workers.dev/coverage`
    );
    expect(await response.text()).toMatchSnapshot();
  });
});
