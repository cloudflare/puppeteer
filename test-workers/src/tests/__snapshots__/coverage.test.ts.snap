// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`browser does basic manipulations > gets the title of google developers website 1`] = `"Bytes used: 20.30696576151122%"`;
